File: TiaCodegen/Commands/Comparisons/CompareOperator.cs
using TiaCodegen.Interfaces;

namespace TiaCodegen.Commands.Comparisons
{
    public abstract class CompareOperator : BaseOperationOrSignal
    {
        public CompareOperator(params IOperationOrSignal[] operationOrSignals)
            : base(operationOrSignals)
        { }
    }
}

File: TiaCodegen-ts/src/Commands/Comparisons/CompareOperator.ts
import { IOperationOrSignal } from '../../Interfaces/IOperationOrSignal';
import { BaseOperationOrSignal } from '../BaseOperationOrSignal';

export abstract class CompareOperator extends BaseOperationOrSignal {
    constructor(...operationOrSignals: IOperationOrSignal[]) {
        super(...operationOrSignals);
    }
}
